contrib/garyburd/redigo/redigo.go

```diff
@@ -68,7 +70,12 @@ func Dial(network, address string, options ...interface{}) (redis.Conn, error) {
 	if err != nil {
 		return nil, err
 	}
-	tc := Conn{c, &params{cfg, network, host, port}}
+	db := 0
+	dialCfg, ok := redigo.ResolveDialOpts(dialOpts)
+	if ok {
+		db = dialCfg.DB
+	}
+	tc := Conn{c, &params{cfg, network, host, port, db}}
 	return tc, nil
```

Comment:
```suggestion
	var db int
	if dialCfg, ok := redigo.ResolveDialOpts(dialOpts); ok {
		db = dialCfg.DB
	}
	return Conn{c, &params{cfg, network, host, port, db}}, nil
```

Reply:
This simplifies and shortens the code a bit. Can you make this change here and throughout?